### ISSUE TITLE
[$40]{MSFT-125} Sample challenge page: Screen reader reads the registrants information in a textual format and not in a tabular format.

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2823]
**Steps to Reproduce**
1. Go to the sample challenge page: https://clicktime.symantec.com/34CaebazimgmFxdkBZnihRc7Vc?u=https%3A%2F%2Fwww.topcoder.com%2Fchallenges%2F<sample-Sample challenge page:challengeid>?tab=details. 2. Ensure that the screen reader is switched on. 3. Navigate to the 'registrants' control using the tab key or the arrow key. 4. Press enter on the 'registrants' control. 6. Press the down arrow to access the registrants information. 7. The information is displayed in a tabular form, however screen reader doesn't recognise the tabular format.

**Expected Result**
Native tabular structure should be used so as the screen reader reads the data in a tabular format.

**Actual Result**
Screen reader reads the registrants information in a textual format and not in a tabular format.

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 1. Perceivable
**WCAG Level**: A
**WCAG Success Criterion**: 1.3.1 Info and Relationships
**Screen Reader Used**: JAWS
**Accessibility Audit Tool used**: N/A

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=088f5f5d%2D9b1c%2D4b7e%2Dacbf%2D320c257c9a9a&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2FKameshwarid%2Fbug%2023

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30095999 has been created for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30095999 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30095999 has been updated - it has been assigned to sadathanwar17.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097274 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097274 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097274 has been updated - it has been assigned to sadathanwar17.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by veshu** [comment]
@asadath1395 This is not correct fix, please see the example here how this should be read https://www.youtube.com/watch?v=Vsj1MIg-gf8

**Comment by asadath1395** [comment]
@veshu Lets open this one

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30097274 has been updated - it has been assigned to rohitgupta_.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by drasticdpk** [comment]
Verified on Windows/Chrome/JAWS
Reference link :
https://support.freedomscientific.com/Training/Surfs-Up/Tables.htm

**Comment by drasticdpk** [comment]
JAWS is saying like "No Table Found!"

**Comment by r0hit-gupta** [comment]
@drasticdpk unable to view my changes in the DOM. May you please verify the deployment? Thanks!

![image](https://user-images.githubusercontent.com/16841686/62521024-6dc28280-b84c-11e9-9e6a-3bca8595fed2.png)

**Comment by veshu** [comment]
@sushilshinde Not deployed to prod?

**Comment by tosha5252** [comment]
Verified working...announces the table

Windows 10 env
Browser: Chrome

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30097274<br/><br/>```This is an automated message for crazyk via Topcoder X```
